### DISCUSSION TITLE
Blog-Website

### DISCUSSION THREAD

**Discussion by the-coder-o** [community/community#108293] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hi, I'm Abdul Basit, a fullSatck developer who loves building cool things with code. In addition to coding, I also make more interesting fullSatck applications, read more from https://a-bd.me

**Comment by queenofcorgis** [comment, marked as answer]
Hi @the-coder-o , we’re glad you’re here 😀

Thanks for sharing your website with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
